src/demo/story.js

```diff
@@ -0,0 +1,24 @@
+import { storiesOf } from '@storybook/html';
+import { html, render } from 'lit-html';
+import Readme from '../../README.md';
+
+const story = storiesOf('Welcome', module);
+story.addParameters({
+  readme: {
+    sidebar: Readme,
+  },
+});
+
+story.add('to Patterns Library', () => {
```

Comment:
`Welcome - to patterns library` looks a bit workaroundy.

I would call the folder "Pattern Library" (without s) and the Story "Home".

Reply:
I saw my variant while googeling... i like it. Can change it if the team wants it.